### ISSUE TITLE
Fabric 1.20.1 Ship doesn't disassemble, and has a wobbly steering wheel glitch when trying to turn

### ISSUE THREAD

**Issue by JudgeBeef** [ValkyrienSkies/Eureka#266]
### This issue occurs when only Valkyrien Skies and addons are installed and no other mods

- [X] I have tested this issue and it occurs when no other mods are installed

### Minecraft Version

1.20​

### Mod Loader

Fabric

### Issue description

After assembling a ship, selecting align, and disassemble, the button doesn't do anything and the ship remains assembled.
Also, if you drive the ship, then stop driving, and then try to drive again, the steering wheel seems to jam up and just wobbles the ship back and forth, and can only be fixed when selecting the align button a couple of times before driving again. 

### Issue reproduction

For disassembly issue:
1. Install Valkyrien Skies and Eureka! Ships! on fabric 1.20.1
2. Create a ship in game
3. Attempt to disassemble ship through the steering wheel

For steering wheel issue:
1. Drive your ship for the first time
2. Stop driving by crouching (left shift)
3. Try to drive again via the steering wheel and turn
4. wobble time

### Logs

[latest.log](https://github.com/ValkyrienSkies/Eureka/files/12991214/latest.log)
unfortunately, there is no debug.log available since there have been no crashes (not yet at least)

**Comment by PumaBlack** [comment]
Temp Fix for the wobble:

I figured out, that the ship becomes unable to turn, after it aligns. To fix this you have to sneak and click the helm again, then press the align button again, and now the wobble is gone... until you align again. But you can repeat the fix.

The disassemble issue on the other hand is still there. 1.20.1 forge on my server.

incredible good mod, but the disassemble "bug" takes away a huge part of that cozy feeling. If you can't disassemble the ship while it rains, it rains everywhere and you can't some mods like More Red or Pretty Pipes proper, while the ship is an entity.

I hope, there will be an update, that fixes the "disassemble-bug"

**Comment by JudgeBeef** [comment]
Yeah I noticed the align fix fairly quickly, but it's still annoying to have to do each time I want to drive my ship. The mod clearly wants to act like a normal Minecraft boat, aligned or not, where you can turn in whatever direction you want. But alas. Wobble.
But yeah, it is a great mod; it's unlike any mod I've ever seen before and it adds so much to the Minecraft world. I hope the devs make it even better by fixing this stuff.

**Comment by PriestOfFern** [comment]
1.16,1.19,1.20.1 versions of eureka just aren't maintained all that much. The last update for a version that wasn't 1.18 was in july.
